cpp/src/text/edit_distance.cu

```diff
@@ -42,91 +44,76 @@ namespace detail {
 namespace {
 
 /**
- * @brief Compute the edit-distance between two strings
+ * @brief Compute the Levenshtein distance for each string pair
  *
- * The temporary buffer must be able to hold 3 int16 values for each character
- * in the smaller of the two provided strings.
+ * Documentation here: https://www.cuelogic.com/blog/the-levenshtein-algorithm
```

Comment:
Is this supposed to have docs?

Reply:
Oops. Maybe the diff I saw was outdated. This review might be totally wrong. Let me refresh and try again.

Reply:
This is just a rendering issue on my side. I could not see the link in my browser. All I saw was "Documentation here:" and missed the link (you can see it's a very slightly different font color). Apologies, this is fine. Approving.

![image](https://github.com/rapidsai/cudf/assets/3943761/be576232-e235-41b5-a2d2-6b170e69c586)